pkg/prometheus/promcfg.go

```diff
@@ -4092,3 +4066,11 @@ func getLowerByteSize(v *monitoringv1.ByteSize, cpf *monitoringv1.CommonPromethe
 func isByteSizeEmpty(v *monitoringv1.ByteSize) bool {
 	return v == nil || *v == ""
 }
+
+func (cg *ConfigGenerator) addFiltersToYaml(cfg yaml.MapSlice, minimumVersion string, filters []monitoringv1alpha1.Filter) yaml.MapSlice {
+	if filters == nil {
```

Comment:
```suggestion
	if len(filters) == 0 {
```

Reply:
Same here, I changed it after. I will change it back.